### PR TITLE
Adding an example using the component name

### LINKED ISSUE #4890: Devdocs should give a better example about the import/export/listen properties
# Clarify the usage of import, export and listen

## Topic Link

https://devdocs.magento.com/guides/v2.3/ui_comp_guide/concepts/ui_comp_linking_concept.html

## Description

I think that it should have another example when we explain inside of a `.js` file, we are just using this type of example:
```javascript
{
  "imports": {
   "visible": "${ $.provider }:visibility"
  }
}
```

## Possible solutions

Add another example, like this one:
```javascript
{
  "imports": {
   "items": "checkout.sidebar.summary.cart_items:items"
  }
}
```

### PR DESCRIPTION
**Pull request by LucasCalazans**
## Purpose of this pull request

This pull request (PR) adds another way to use export, import and listen

## Affected DevDocs pages

v2.3:
https://devdocs.magento.com/guides/v2.3/ui_comp_guide/concepts/ui_comp_linking_concept.html

v2.2
https://devdocs.magento.com/guides/v2.2/ui_comp_guide/concepts/ui_comp_linking_concept.html

## Links to Magento source code

https://github.com/magento/devdocs/blob/master/guides/v2.2/ui_comp_guide/concepts/ui_comp_linking_concept.md

## Issue

Closes #4890

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/ui_comp_guide/concepts/ui_comp_linking_concept.md [hunk 1]
```diff
@@ -33,9 +33,23 @@ Example of setting `exports` in a component's `.js` file:
   }
 }
 ```
-
 Here `visible` is the `key`, `${ $.provider }:visibility` is the `value`. The value of the local `visible` property is assigned to the `visibility` property of the `provider` component. The latter is changed automatically if the value of `visible` changes if the local `visible` property is observable (which it isn't given only the code example above).
 
+If you want to use directly, you can use the destination component name:
```
Comment (erikmarr) [inline guides/v2.2/ui_comp_guide/concepts/ui_comp_linking_concept.md]:
```suggestion
Example of setting `exports` directly using the destination component name:
```

File: guides/v2.2/ui_comp_guide/concepts/ui_comp_linking_concept.md [hunk 2]
```diff
@@ -33,9 +33,23 @@ Example of setting `exports` in a component's `.js` file:
...
+
+```json
+{
+  "exports": {
+   "items": "checkout.sidebar.summary.cart_items:items"
+  }
+}
+```
+
+This name is created by the hierarchy in the XML handle. Put all parents name together with "." (dot) and, at the end, add the component name.
```
Comment (erikmarr) [inline guides/v2.2/ui_comp_guide/concepts/ui_comp_linking_concept.md]:
```suggestion
The syntax for the destination component name is determined by the hierarchy in the XML handle. Separate parent names with a "." (dot) followed by the component name.
```

File: guides/v2.2/ui_comp_guide/concepts/ui_comp_linking_concept.md [hunk 3]
```diff
@@ -33,9 +33,23 @@ Example of setting `exports` in a component's `.js` file:
...
+
+{:.bs-callout .bs-callout-info}
+To easily catch the full name, open your browser devtools, select the element that you want on the "Elements" tab, go to the "Console" tab and execute the following code: require('ko').contextFor($0).$data.name.
```
Comment (erikmarr) [inline guides/v2.2/ui_comp_guide/concepts/ui_comp_linking_concept.md]:
```suggestion
To retrieve the full name of the destination component name, open your browser in developer mode, select the element that you want on the **Elements** tab, go to the **Console** tab, and execute the following code: `require('ko').contextFor($0).$data.name`.
```

**Comment by erikmarr** [comment]
@LucasCalazans please update your PR description to indicate that this change also affects the 2.2 version of this topic in devdocs.

**Comment by LucasCalazans** [comment]
@erikmarr PR updated.

File: guides/v2.2/ui_comp_guide/concepts/ui_comp_linking_concept.md [hunk 4]
```diff
@@ -33,9 +33,23 @@ Example of setting `exports` in a component's `.js` file:
...
+
+```json
+{
+  "exports": {
+   "items": "checkout.sidebar.summary.cart_items:items"
+  }
+}
+```
+
+The syntax for the destination component name is determined by the hierarchy in the XML handle. Separate parent names with a "." (dot) followed by the component name.
```
Comment (erikmarr) [inline guides/v2.2/ui_comp_guide/concepts/ui_comp_linking_concept.md]:
```suggestion
The syntax for the destination component name is determined by the hierarchy in the XML handle. Separate parent names with a `.` (dot) followed by the component name.
```

**Comment by LucasCalazans** [comment]
No problem @erikmarr 
Done.

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
Thanks @LucasCalazans!

**Pull request merged**